src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/SimpleConfigModel.cs

```diff
@@ -846,7 +846,7 @@ private List<FileSourceMatchInfo> EvaluateSources(IParameterSet parameters, IVar
                 IReadOnlyList<string> topCopyOnlyPattern = JTokenToCollection(source.CopyOnly, SourceFile, CopyOnlyPatternDefaults).ToList();
                 FileSourceEvaluable topLevelPatterns = new FileSourceEvaluable(topIncludePattern, topExcludePattern, topCopyOnlyPattern);
 
-                Dictionary<string, string> fileRenames = new Dictionary<string, string>(source.Rename ?? RenameDefaults);
+                Dictionary<string, string> fileRenamesFromSource = new Dictionary<string, string>(source.Rename ?? RenameDefaults);
```

Comment:
Not certain if the StringComparer is taken from the original dictionary or if we can still pass it explicitly here

Reply:
source.Rename doesn't have an explicitly defined comparer on it, so passing in explicitly.